src/Facebook/FacebookClient.php

```diff
@@ -162,7 +162,11 @@ public function getBaseGraphUrl($postToVideoUrl = false)
      */
     public function prepareRequestMessage(FacebookRequest $request)
     {
-        $postToVideoUrl = $request->containsVideoUploads();
+        $pathElements = explode('/', $request->getUrl());
+        $lastInPath = end($pathElements);
+        $postToVideoUrl = $request->containsVideoUploads() ||
+            ($lastInPath == 'videos' && $request->getMethod() === 'POST');
```

Comment:
The reason is in the start phase, there is no video files attached, but we need to post to graph-video. 
curl \
  -X POST \
  "https://graph-video.facebook.com/v2.3/1533641336884006/videos"  \
  -F "access_token=XXXXXXXXX" \
  -F "upload_phase=start" \
  -F "file_size=152043520"